R/htmlA.R

```diff
@@ -15,4 +15,5 @@ htmlA <- function(children=NULL, id=NULL, n_clicks=NULL, n_clicks_timestamp=NULL
     component$props <- filter_null(component$props)
 
     structure(component, class = c('dash_component', 'list'))
-}
\ No newline at end of file
+}
+
```

Comment:
are these extra trailing lines intentional? It's nice to have one trailing newline, but 2 is unusual.

Reply:
fixed in ec6fd2b